Automattic/jetpack
# Fix/jetpack newsletter settings endpoint

This PR fixes the settings endpoint response when the `subscription_options` are sent to the endpoint but they are not modified. 

This PR intends to fix the error that shows up when saving the newesletter settings in calypso. 


## Proposed changes:

* Ignore the saving and the false error when the options that current exists match the options in the table. 


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
Apply this patch. 
Go to wordpress.com/settings/newsletter/example.com and make a change to any option beside the welcome text box and notice that you don't see the error any more.
